requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,50 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [x] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
+- [x] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
+- [x] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/pull/8858) - Marketing
+- [ ] Create dashboard templates for 2x common usecases - @ivanagas
```

Comment:
@ivanagas I see we've got one ready, but not sure if it's internal or not. Just wanted to check if we should delete this, or not?

Reply:
I should have the first one ready today, but just internal at the moment.